docs/src/pages/components/stack/InteractiveStack.js

```diff
@@ -0,0 +1,196 @@
+import * as React from 'react';
+import FormControl from '@material-ui/core/FormControl';
+import FormLabel from '@material-ui/core/FormLabel';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import Grid from '@material-ui/core/Grid';
+import HighlightedCode from 'docs/src/modules/components/HighlightedCode';
+import Paper from '@material-ui/core/Paper';
+import RadioGroup from '@material-ui/core/RadioGroup';
+import Radio from '@material-ui/core/Radio';
+import Stack from '@material-ui/core/Stack';
+
+export default function InteractiveStack() {
+  const [direction, setDirection] = React.useState('row');
+  const [justifyContent, setJustifyContent] = React.useState('center');
+  const [alignItems, setAlignItems] = React.useState('center');
+  const [spacing, setSpacing] = React.useState(1);
+
+  const jsx = `
+<Stack
+  direction="${direction}"
+  justifyContent="${justifyContent}"
+  alignItems="${alignItems}"
+  spacing={${spacing}}
+>
+`;
+
+  return (
+    <Stack sx={{ flexGrow: 1 }}>
+      <Stack
+        direction={direction}
+        justifyContent={justifyContent}
+        alignItems={alignItems}
+        spacing={spacing}
+        sx={{ height: 240 }}
+      >
+        {[0, 1, 2].map((value) => (
+          <Paper
+            key={value}
+            sx={(theme) => ({
+              padding: theme.spacing(2),
+              paddingTop: (value + 1) * 10,
+              paddingBottom: (value + 1) * 10,
+              color: theme.palette.text.secondary,
+            })}
+          >
+            {`Cell ${value + 1}`}
+          </Paper>
+        ))}
+      </Stack>
+      <Paper sx={(theme) => ({ padding: theme.spacing(2) })}>
+        <Grid container spacing={3}>
+          <Grid item xs={12}>
+            <FormControl component="fieldset">
+              <FormLabel>direction</FormLabel>
```

Comment:
`fieldset > label`, this seems odd. I was either expecting `fieldset > legend` or `div > label`.

Reply:
Oops, yeah I'll clean this up. It was copied from the `Grid` demo 😇